pkg/process/config/config.go

```diff
@@ -363,12 +366,14 @@ func loadEnvVariables() {
 	}
 
 	// Support API_KEY and DD_API_KEY but prefer DD_API_KEY.
-	if key, ddkey := os.Getenv("API_KEY"), os.Getenv("DD_API_KEY"); ddkey != "" {
-		log.Info("overriding API key from env DD_API_KEY value")
-		config.Datadog.Set("api_key", strings.TrimSpace(strings.Split(ddkey, ",")[0]))
-	} else if key != "" {
-		log.Info("overriding API key from env API_KEY value")
-		config.Datadog.Set("api_key", strings.TrimSpace(strings.Split(key, ",")[0]))
+	apiKey, envKey := os.Getenv("DD_API_KEY"), "DD_API_KEY"
```

Comment:
Looks like the exact same thing from above?

Reply:
Yeah, the previous logic was a bit unwieldy to read so I left this cleanup in.